pt-BR.json

```diff
@@ -409,8 +409,8 @@
 			"add-to-dictionary": "Adicionar ao dicionário"
 		},
 		"heading-suggestion": {
-			"label-no-heading": "No heading",
-			"label-heading-level": "Heading {{level}}"
+			"label-no-heading": "Sem cabeçalho",
```

Comment:
There is a different translation from the `pt-BR.termbase.md` "heading" term used here. That's because the term used here refers to the heading element itself (H1-H6) while the term written on termbase refers to the whole section under a heading element.